src/drivers/voxl2_io/voxl2_io.cpp

```diff
@@ -248,27 +242,30 @@ int Voxl2IO::get_version_info()
 				} else {
 					_bytes_sent+=cmd.len;
 					_packets_sent++;
-					px4_usleep(2000);
 				}
 				break;
 
+			/* We got a valid response! Update version info */
 			}  else {
 				memcpy(&_version_info, &_read_buf[header], sizeof(VOXL2_IO_VERSION_INFO));
 				read_succeeded = 1;
 				break;
 			}
+
+		/* We didn't get a response from Voxl2 IO board, try again... */
 		} else {
 			read_retries--;
+			PX4_ERR("Failed to receive version info, %i retries left...", read_retries);
```

Comment:
Is this really an error?

Reply:
This has been updated in commit [d69b600](https://github.com/modalai/px4-firmware/pull/50/commits/d69b60038b27befb45d73bd44bf01b56d99f2bc8).